test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -283,6 +283,11 @@ public void BinaryClassifierLogisticRegressionTest()
         [Trait("Category", "SkipInCI")]
```

Comment:
❓  I guess I am missing something... but shouldn't this line be removed in order to check that the updated baselines are actually correct?

Reply:
These baselines haven't changed in the master branch since I first implemented this pull request. I'm assuming the new values (which were correct then) are still correct now. They would be validated as part of #4722.